roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -289,6 +290,9 @@
     state: present
     name: "logging-{{ es_component }}-cluster"
     namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    clusterip: 'None'
+    annotations:
+      service.alpha.kubernetes.io/tolerate-unready-endpoints: 'true'
```

Comment:
what does this annotation do?

Reply:
This tells kubernetes to publish the pod endpoints regardless if the readiness probe has yet succeed. My initial tests showed the ES nodes were unable to find each other via the cluster service because there were no published endpoints.  This leads to the discovery failing and nodes in the cluster never finding each other.